package.json

```diff
@@ -50,6 +50,7 @@
   "devDependencies": {
     "babel-core": "^6.26.0",
     "babel-loader": "^7.1.2",
+    "babel-polyfill": "^6.26.0",
```

Comment:
why is the babel polyfill needed?

Reply:
I don't  remember very clear,  it seems like Object.keys in early Chrome  not support traversing characters